pkg/users/account-create-dialog.js

```diff
@@ -199,10 +273,15 @@
         if (field == "locked")
             state.change_passw_force = false;
 
+        if (field == "home_dir") {
+            state.home_dir_dirty = true;
+            state.home_dir_exists = false;
+        }
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230202-125519-871b841d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)